SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -923,7 +927,13 @@ void FSpatialGDKEditorToolbarModule::LaunchInspectorWebpageButtonClicked()
 	}
 
 	FString WebError;
-	FPlatformProcess::LaunchURL(*InspectorURL, TEXT(""), &WebError);
+	if (FSpatialPackageManager::TryFetchInspectorBinary()) {
+		FPlatformProcess::LaunchURL(*InspectorURL, TEXT(""), &WebError);
```

Comment:
Perhaps it would be better to move WebError and its processing to this `if` section?

Reply:
I still think `FString WebError;` should in the `if` section.

Reply:
oh, yes, we thought you were referring to the `if(!WebError.IsEmpty())` part